dbms/src/Formats/ODBCDriver2BlockOutputStream.cpp

```diff
@@ -0,0 +1,102 @@
+#include <Core/Block.h>
+#include <Formats/FormatFactory.h>
+#include <Formats/ODBCDriver2BlockOutputStream.h>
+#include <IO/WriteBuffer.h>
+#include <IO/WriteHelpers.h>
+
+
+#include <Core/iostream_debug_helpers.h>
+
+
+namespace DB
+{
+ODBCDriver2BlockOutputStream::ODBCDriver2BlockOutputStream(
+    WriteBuffer & out_, const Block & header_, const FormatSettings & format_settings)
+    : out(out_), header(header_), format_settings(format_settings)
+{
+}
+
+void ODBCDriver2BlockOutputStream::flush()
+{
+    out.next();
+}
+
+void writeODBCString(WriteBuffer & out, const std::string & str)
+{
+    writeIntBinary(Int32(str.size()), out);
+    out.write(str.data(), str.size());
+}
+
+void writeODBCString(WriteBuffer & out, const char * cstr)
```

Comment:
No need for this method. It will work automatically by implicit type conversion.

Reply:
it does not work for writeODBCString(out, "woow")